test/datatypes/datetime.coffee

```diff
@@ -1618,12 +1641,12 @@ describe 'DateTime.after', ->
     DateTime.parse('2000-02-15T12:30:30.500+00').after(DateTime.parse('2000-02-15T12:30:30.0+00'), DateTime.Unit.MILLISECOND).should.be.true()
 
   it 'should return null in cases where a is b but there are unknown values', ->
-    should.not.exist DateTime.parse('2000-01-01T00:00:00').after(DateTime.parse('2000-01-01T00:00:00'))
-    should.not.exist DateTime.parse('2000-01-01T00:00').after(DateTime.parse('2000-01-01T00:00'))
-    should.not.exist DateTime.parse('2000-01-01T00').after(DateTime.parse('2000-01-01T00'))
-    should.not.exist DateTime.parse('2000-01-01').after(DateTime.parse('2000-01-01'))
-    should.not.exist DateTime.parse('2000-01').after(DateTime.parse('2000-01'))
-    should.not.exist DateTime.parse('2000').after(DateTime.parse('2000'))
+    DateTime.parse('2000-01-01T00:00:00').after(DateTime.parse('2000-01-01T00:00:00')).should.be.false()
```

Comment:
The test case description needs to be updated since this no longer returns `null`.